examples/amplify-react/README.md

```diff
@@ -0,0 +1,117 @@
+
+An example project for KVS WebRTC. This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app) and was built to work with [Amplify](https://docs.amplify.aws/). AWS services that this project integrates with are Cognito for user authentication and authorization, and Amazon Kinesis Video Streams for WebRTC.
+
```

Comment:
link to setup Cognito https://docs.aws.amazon.com/cognito/latest/developerguide/what-is-amazon-cognito.html  ?